e2e-tests/tests/headlamp.spec.ts

```diff
@@ -16,3 +16,13 @@ test("headlamp is there and so is minikube", async ({ page }) => {
 
   await expect(page).toHaveURL(/.*main/);
 });
+
+test('GET /plugins/list returns plugins list', async ({ page }) => {
+  const response = await page.goto('/plugins/list');
+  expect(response).toBeTruthy();
+
+  const text = await response.text();
+
+  expect(text).toBeTruthy();
+  expect(text.length).toBeGreaterThan(0);
```

Comment:
The endpoint returns an array of strings, when the array is empty it will return `[]` and the text length > 0 will still pass, the test case can be improved.

Reply:
Sure, make sense.